### DISCUSSION TITLE
query after version 0.21.3

### DISCUSSION THREAD

**Discussion by benisraelnir** [tree-sitter/py-tree-sitter#275] [Q&A] [answered]
With latest tree-sitter for python (https://pypi.org/project/tree-sitter/) or any version after 0.21.3
I wondered what is the best practice for query.
This what worked for me on 0.21.3:
please assume the code is in code and my query is in my_query

        lang='cpp'
        language = get_language(lang)
        parser = get_parser(lang)
        tree = parser.parse(bytes(code, "utf-8"))
        query = language.query(my_query)
        captures = query.captures(tree.root_node)
This is no longer work as get_language expect an object?
Anyhow, How should I do it?

**Comment by ObserverOfTime** [comment, marked as answer]
Install tree-sitter-cpp etc. instead of tree-sitter-languages.
